values.yaml

```
@@ -1,10 +1,11 @@
 # App specific configs
 scale:
+  enabled: true
   minReplicas: 1
   maxReplicas: 10
   minAvailable: 50%
   cpuThresholdPercentage: 60
```

I would suggest the following change:
```suggestion
  cpuThresholdPercentage: 100
```
as a better default. Setting the threshold lower than the request causes pods to scale before using the resources they requested. Therefore we have a lot of requested resources not used (i.e. wasted) on each node. In my opinion, by default,  pods should start scaling when the min number of replicas is very close to the requested resources. Keep in mind that for special cases this can still be set differently.